### PR TITLE
Derrick/full create trip integration

### PR DESCRIPTION
**Pull request by derrick-yyang**
# Changes

The changes here updated the SessionCode page to display the proper session code, and also populate the tripData in the UserViewModel for the user that just created the trip

### CODE REVIEW

File: frontend-android/app/src/main/java/com/example/wetravel/models/DataModels.kt [hunk 1]
```diff
@@ -24,12 +24,17 @@ enum class VotingPhase {
     RESULTS
 }
 
+data class TripUsers(
+    val userID: String = "",
+    val votes: Int = 0,
+)
+
```
Comment (jasssse) [inline frontend-android/app/src/main/java/com/example/wetravel/models/DataModels.kt:31]: How did we want to keep track of the places a user voted for?

Comment (jasssse) [reply frontend-android/app/src/main/java/com/example/wetravel/models/DataModels.kt:31]: and how many votes they casted for it?

**Comment by derrick-yyang** [comment]
also i need everyones opinion on this, but currently for my PR, tripData is the ENTIRE trip payload, it's probably best to separate this into diff viewmodel vars right? And load these vars individually, so we dont have to keep digging the tripData properties throughout the code.

Also asking cuz loadTrip will affect have conflicts with everyone working on fe integration, since im defining the vars being used

**Review by jerry-cheng5 — COMMENTED** [review]
I think if tripData is just the data from one trip, we dont need to further separate it.

One case where it might be beneficial to separate it would be when we load trip info on the landing page, since we only need basic info like name, location, and num of participants. Though I may make another endpoint to get like a tripSummary or smth.

**Pull request merged**